src/phpDebug.ts

```diff
@@ -458,41 +462,43 @@ class PhpDebugSession extends vscode.DebugSession {
                 // for all connections
                 await Promise.all(
                     connections.map(async (connection, connectionIndex) => {
-                        // clear breakpoints for this file
-                        // in the future when VS Code returns the breakpoint IDs it would be better to calculate the diff
-                        const { breakpoints } = await connection.sendBreakpointListCommand()
-                        await Promise.all(
-                            breakpoints
-                                // filter to only include line breakpoints for this file
-                                .filter(
-                                    breakpoint =>
-                                        breakpoint instanceof xdebug.LineBreakpoint &&
-                                        isSameUri(fileUri, breakpoint.fileUri)
-                                )
-                                // remove them
-                                .map(breakpoint => breakpoint.remove())
-                        )
-                        // set new breakpoints
-                        await Promise.all(
-                            xdebugBreakpoints.map(async (breakpoint, index) => {
-                                try {
-                                    await connection.sendBreakpointSetCommand(breakpoint)
-                                    // only capture each breakpoint once
-                                    if (connectionIndex === 0) {
```

Comment:
I simplified the code and noticed that you removed these `if` checks. Was that intentional?

Reply:
@felixfbecker Yes, because connectionIndex of 0 could be the connection that is waiting for PHP code to execute, connectionIndex 1 would be the "new" connection that is able to execute right away so is able to set the value in time for it to matter.

Plus all it is doing is setting a value and the value is fixed, it doesn't save time by skipping it if already set.